data/json/items/gun/223.json

```diff
@@ -432,6 +432,36 @@
       }
     ]
   },
+  {
+    "id": "m4_survival_rifle",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "M4 Survival Rifle" },
+    "description": "A small game rifle issued to pilots in the 1940s, the M4 Survival Rifle is designed to hunt small game.  Originally chambered in .22 Hornet, it looks like someone rechambered it for 5.56.  The rifle is small, easy to use, and has little kick, but it has little stopping power and a tiny 4-round magazine.",
+    "weight": "1814 g",
+    "volume": "700 ml",
+    "longest_side": "330 mm",
+    "price": 75000,
+    "price_postapoc": 1250,
+    "to_hit": 1,
+    "bashing": 6,
+    "material": [ "steel" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "223" ],
+    "ranged_damage": { "damage_type": "stab", "amount": -2 },
+    "dispersion": 110,
+    "durability": 12,
+    "min_cycle_recoil": 500,
+    "default_mods": [ "factory_handguard" ],
```

Comment:
No built-in  collapsing wire stock? 🤔

Reply:
Would you rather I use .22 LR?  I can do that, or I can try to make .22 Hornet as a separate ammo, but this is my first go 'round, and I don't know how that'll work.